apps/client/src/widgets/onboarding/components/dropdown/dropdown.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import { Icon } from '@bds/ui/icons';
+
+import OptionItem from './option-item';
+
+import * as styles from './dropdown.css';
+
+const OPTIONS = [
+  '사무',
+  '교육/전문',
+  '서비스',
+  '운송/배달',
+  '생산/기술',
+  '예술/프리랜서',
+  '학생/구직자',
+  '해당없음(기타)',
+];
+
+const DEFAULT_PLACEHOLDER = '직업을 선택해주세요.';
+
+const DropDown = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selected, setSelected] = useState<string | null>(null);
+
+  const handleOptionClick = (option: string) => {
+    setSelected(option);
+    setIsOpen(false);
+  };
+
+  const handleToggleDropdown = () => {
+    setIsOpen((prev) => !prev);
+  };
+
+  return (
+    <div className={styles.dropdownWrapper}>
+      <div
+        className={
+          isOpen === false
```

Comment:
```suggestion
          !isOpen 
```

똑같은 의미같아용

Reply:
```suggestion
className={!isOpen ? styles.dropdownContainer : styles.dropdownContainerOpen}
```

코드 제안 드릴게요 최종적으로 이렇게 삼항 연산자를 사용하는 것 어떨까요? 나중에 clsx 를 도입하도록 하죠..